db/migrate/20240108150948_add_sort_index_to_course.rb

```diff
@@ -1,5 +1,13 @@
 class AddSortIndexToCourse < ActiveRecord::Migration[7.0]
+  class Course < ApplicationRecord
+  end
+
   def change
     add_column :courses, :sort_index, :integer, default: 0
+
+    Course
```

Comment:
@yash-learner It doesn't, actually. Regardless of whether it's scoped by school, the non-contiguous numbers in a school's courses' `sort_index` numbers will still result in the courses being shown in the correct sort order.

Because of this, we can simply wait for a school admin to manually sort the courses at some point to fix the `sort_index` to hold only contiguous numbers.

Reply:
@harigopal Yes, you're right, when the user makes first swap all indexes will be in sequential.